composer.json

```diff
@@ -40,23 +40,23 @@
         "ext-simplexml": "*",
         "fidry/cpu-core-counter": "^0.4.1 || ^0.5.1",
         "jean85/pretty-package-versions": "^2.0.5",
-        "phpunit/php-code-coverage": "^10.0.2",
+        "phpunit/php-code-coverage": "^10.1.0",
         "phpunit/php-file-iterator": "^4.0.1",
         "phpunit/php-timer": "^6.0",
-        "phpunit/phpunit": "^10.0.17",
-        "sebastian/environment": "^6.0",
-        "symfony/console": "^6.2.7",
-        "symfony/process": "^6.2.7"
+        "phpunit/phpunit": "10.1.x-dev",
```

Comment:
10.1.0 just released 👍 https://github.com/sebastianbergmann/phpunit/releases/tag/10.1.0 so it can directly use the version